.github/workflows/docs.yml

```diff
@@ -23,6 +23,20 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Check copyright line
+        shell: bash
+        run: |
+          EXIT_CODE=0
+
+          while IFS= read -r -d '' file; do
+              COPYRIGHT=$(grep -h -r "copyright = " "$file")
+              if [ "$COPYRIGHT" != "copyright = f\"{datetime.date.today().year} Flower Labs GmbH\"" ]; then
+                  echo "::error file=$file::Wrong copyright line. Expected: copyright = f\"{datetime.date.today().year} Flower Labs GmbH\""
+                  EXIT_CODE=1
+              fi
+          done < <(find . -path "*/doc/source/conf.py" -print0)
+
+          exit $EXIT_CODE
```

Comment:
@Robert-Steiner could you move the test code into a global `./dev/test-copyright.sh` file and invoke that one here? We intend to bundle the invocation of all current and future copyright-related tests in one file. Apart from this one thing, everything looks good!

Reply:
I will move it there 👍